### ISSUE TITLE
Peer review from Ellen

### ISSUE THREAD

**Issue by ellenhsieh** [css-research/fp-hanjiaxu#4]
Thank you for presenting such interesting topic on identifying risk factors for early-onset depression. The outlay of the poster is pretty neat and clean. It is nice that a lot of plots and tables are included in this poster to present the information instead of just using texts. 

As for the design of this poster, the color is really bright and comfortable for audience to read. Some of the plots’ labels are cut, it would be nicer if you can save the image more completely. The table of comparing different models are really clear and using the red color to let the best model stand out is really clever. The ICE/PDP plots seems really great and easy to read the relationship between the predictors and dependent variable. 

The arrangement of the text in each section looks nice overall, especially for conclusion and reference, using the bullet points is a great idea. Only the texts in introduction seems too crowded. Maybe it would be nice to use a bigger font for the whole poster and make the background introduction more concise. With bigger font size, it would be easier for audience to read the contents. It is a really nice idea to use the procedure graph to make the process clearer for audience to understand and get the big picture. Nevertheless, it would nicer to make this graph bigger so that it would be easy to read on the poster.

To sum up, this is a really nice poster to presenting your study and interested topic. Everything is clear and easy to get the idea of your research.
